src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -61,6 +63,15 @@ public OpenLoadFlowParameters setDc(boolean dc) {
         return this;
     }
 
+    public boolean hasGeneratorVoltageRemoteControl() {
+        return generatorVoltageRemoteControl;
+    }
+
+    public OpenLoadFlowParameters setGeneratorVoltageRemoteControl(boolean generatorVoltageRemoteControl) {
```

Comment:
Can we changed this parameter from the config.yml file ? It is required for me if I want to use the same tool (build with ittols packager) to validate LF results from different LF engines. Can we create a section ````openloadflow-default-parameters````. Or maybe this parameter should be located in the common loadflow API.

Reply:
Yes, this parameter should be in common loadflow parameters as many other one (distributed slack, etc).
We should open an issue and list parameters (from open load flow and hades2) that could be shared  between all implementations.
Persistence of default values already exists for common loadflow parameters so we could do it here once the generatorVoltageRemoteControl have been moved.